.coveragerc

```diff
@@ -0,0 +1,5 @@
+# .coveragerc to control coverage.py
```

Comment:
We don't need this

Reply:
not a problem -- my intention was to not have the unit test sources be considered when calculating code coverage.   However, if one looks more at the detailed list  and less at the over all number, then the information is approximately the same either way (implicitly versus explicitly ignoring the test files).